src/db.js

```
@@ -22,11 +22,11 @@ const {
 
 const sequelize = new Sequelize({
   database: POSTGRES_DB || 'postgres',
-  username: POSTGRES_USER || 'postgres',
+  username: POSTGRES_USER || 's-stanishevska',
```

This line sets a hard-coded default username 's-stanishevska'. That looks like a developer-specific value. If other contributors or CI use a different default DB user, the connection may fail when POSTGRES_USER is not set. Consider removing project-specific defaults or changing to a more common default (e.g., 'postgres') and document required env variables in a .env.example so tests run reliably locally.